File: Modul 1/06. JavaScript Basics/10. Exam-Preps/JavaScript - 1 April 2013 - Morning/Problem 1/Solution.cs
using System;

class Solution
{
    static void Main()
    {
        int N = int.Parse(Console.ReadLine());
        int count = 1;
        int last = int.Parse(Console.ReadLine());
        for (int i = 1; i < N; i++)
        {
            int current = int.Parse(Console.ReadLine());
            if (current < last)
            {
                count++;
            }
            last = current;
        }

        Console.WriteLine(count);
    }
}

File: Modul 1/06. JavaScript Basics/10. Exam-Preps/JavaScript - 1 April 2013 - Morning/Problem 1/Solution.js
function Solve(params) {
	var N = parseInt(params[0]);
	var count = 1;
	var last = parseInt(params[1]);
	for (var i = 2; i <= N; i++) {
		var current = parseInt(params[i]);
		if (current < last) {
			count++;
		}
		last = current;
	}
	return count;
}
